purchase_order_line_deep_sort/tests/test_purchase_order_line_deep_sort.py

```diff
@@ -0,0 +1,158 @@
+# Copyright 2018 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests import common
+
+
+class TestPurchaseOrderLineDeepSort(common.SavepointCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestPurchaseOrderLineDeepSort, cls).setUpClass()
+
+        po_model = cls.env['purchase.order']
+        cls.po_line_model = cls.env['purchase.order.line']
+        cls.product_1 = cls.env['product.product'].create({
```

Comment:
here `cls.env['product.product']` and other objects use multiple time, so use as a global variable.

Reply:
This is not adding too much value in terms of performance, but we'll change it if we have to touch again the module.